### DISCUSSION TITLE
G6 sensor date not the same

### DISCUSSION THREAD

**Discussion by robertrub** [NightscoutFoundation/xDrip#3288] [Bug reports] [answered]
Minor bug. I use 2 G6 transmitters (the _legal_ one and Anubis. xDrip Master had the _legal_ transmitter and sensor 1. xDrip variant1 has Anubis and sensor 2. It started on day 9 of the first sensor.

_Legal_ transmitter stops at day 10 and master xDrip goes on _Sensor stopped_ mode.

I enter the Anubis serial in Master and _legal_ serial in variant1. 

I need to _start sensor_ in master in order to see BGs. That's not a problem for the Anubis transmitter as it knows that it has already started.

Once "started" it transmits the correct start time to xDrip. On the 2nd _status_ page, all is good.

![Screenshot_20240111_153627_xDrip+.jpg](https://github.com/NightscoutFoundation/xDrip/assets/76626474/826cf25b-282c-447f-8d19-20564e2ace4b)

But, on the first _Status_ page, xDrip still shows the "new" start time

![Screenshot_20240111_153713_xDrip+.jpg](https://github.com/NightscoutFoundation/xDrip/assets/76626474/d35635bb-43e5-4258-a864-a7fbd56f3160)

And, xDrip sends that erroneous 2nd start time to AAPS too. 

![Screenshot_20240111_153643_AAPS.jpg](https://github.com/NightscoutFoundation/xDrip/assets/76626474/98562f33-3654-4345-b1c6-4b44cb1219f7)

It would be better and more accurate if the _start time_ sent from the transmitter was used everywhere.

**Comment by Navid200** [comment, marked as answer]
This PR fixes that.  https://github.com/NightscoutFoundation/xDrip/pull/3263

**Comment by robertrub** [comment]
Great. Thank you. I'll update to the latest of the latest ;) I have the 2023-12-23 version.

**Reply by Navid200** [reply]
The PR has not been reviewed yet.
What I meant was that after that PR is approved and merged, it will be fixed.

**Comment by robertrub** [comment]
Yep. Guessed that when I upgraded to latest (Jan 10th) and the problem was not solved. Then, went into searching the commits and saw that I needed to wait a bit more ;)

**Reply by robertrub** [reply]
Did your PR got merged ? Still not the same time on both pages in xDrip (latest nightly)

![Screenshot_20240229_120523_xDrip+.jpg](https://github.com/NightscoutFoundation/xDrip/assets/76626474/330af3db-63ae-41fc-ae17-c002a90dd4d9)

![Screenshot_20240229_120457_xDrip+.jpg](https://github.com/NightscoutFoundation/xDrip/assets/76626474/3987ea26-3a86-4ad9-b232-b816882b9e2e)

**Reply by Navid200** [reply]
No, it has not been merged.

If you open a PR and you see "Open", it means it has not been merged.
If you see "Merged", it means it has been merged.
If you see "Closed", it means the PR was closed without being merged.
